include/rpt/detector.h

```diff
@@ -0,0 +1,99 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2021 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+
+ *
+ * Author: Audrey Collard-Daigneault, Polytechnique Montreal, 2021-
+ */
+
+#ifndef lethe_detector_h
+#define lethe_detector_h
+
+/**
+ * Contains all properties of a detector
+ */
+
+#include <deal.II/base/point.h>
+
+#include <rpt/parameters_rpt.h>
+
+using namespace dealii;
+
+template <int dim>
+class Detector
+{
+public:
+  /**
+   * @brief Constructor for the Detector.
+   *
+   * @param detector_parameters All parameters related to the detector
+   *
+   * @param n ID number to the particle
```

Comment:
```suggestion
   * @param n ID number of the particle
```
Is this  the id of the detector or the particle here?

Reply:
Typo, it's "of the detector" thanks